setup.py

```diff
@@ -1,28 +1,13 @@
-# pylint: disable=import-error,no-name-in-module
-# See https://github.com/PyCQA/pylint/issues/73#issuecomment-163171888
-from distutils.core import setup
-from distutils.command.install_scripts import install_scripts as _install_scripts
-from distutils.file_util import move_file
+from setuptools import setup
 
-import os
-
-class install_scripts(_install_scripts):
-    def run(self):
-        _install_scripts.run(self)
-
-        for old in self.get_outputs():
-            new = os.path.splitext(old)[0]
-            # move_file doesn't have a way to tell it to overwrite, so remove
-            # the existing copy to accomplish the same thing.
-            if os.path.isfile(new) and os.path.exists(new):
-                os.unlink(new)
-            move_file(old, new)
-
-setup(cmdclass={"install_scripts": install_scripts},
-      name='pykickstart', version='3.27',
+setup(name='pykickstart', version='3.27',
       description='Python module for manipulating kickstart files',
       author='Chris Lumens', author_email='clumens@redhat.com',
       url='http://fedoraproject.org/wiki/pykickstart',
+      install_requires=['six', 'ordered-set', 'requests'],
```

Comment:
Remove `ordered-set` and rebase it on master after I push #330 and I think this one will be good.